File: LeetCode Solutions/Problems/1884. Egg Drop With 2 Eggs and N Floors.py
class Solution:
    def twoEggDrop(self, n: int) -> int:
        minimumNumberOfMoves = 0
        iterable = n
        while iterable > 0:
            minimumNumberOfMoves += 1
            iterable -= minimumNumberOfMoves
        return minimumNumberOfMoves

File: LeetCode Solutions/Problems/1884. Egg Drop With 2 Eggs and N Floors.c


int twoEggDrop(int n){
    int minimumNumberOfMoves = 0;
    int iterable = n;
    while (iterable > 0){
        minimumNumberOfMoves += 1;
        iterable -= minimumNumberOfMoves;
    }
    return minimumNumberOfMoves;
}